executor/oci/user_unix.go

```diff
@@ -0,0 +1,43 @@
+// +build !windows
+
+package oci
+
+import (
+	"context"
+
+	"github.com/opencontainers/runc/libcontainer/user"
+)
+
+func GetUser(ctx context.Context, root, username string) (uint32, uint32, []uint32, error) {
```

Comment:
What/why is unix specific in this file. Seems to be functions that just do text parsing.

Reply:
GetPasswdPath and GetGroupPath are not defined in github.com/opencontainers/runc/libcontainer/user

Reply:
These are constants anyway. Can be redefined.